app/models/comment.rb

```diff
@@ -0,0 +1,12 @@
+class Comment < ApplicationRecord
```

Comment:
All expected models, User, Post, Comment, and Like are properly added ✔ All models and their associations are correct 🙌